api/src/test/java/org/pmiops/workbench/profile/ProfileServiceTest.java

```diff
@@ -31,22 +35,52 @@
 import org.pmiops.workbench.institution.VerifiedInstitutionalAffiliationMapper;
 import org.pmiops.workbench.institution.VerifiedInstitutionalAffiliationMapperImpl;
 import org.pmiops.workbench.institution.deprecated.InstitutionalAffiliationMapperImpl;
+import org.pmiops.workbench.model.Address;
 import org.pmiops.workbench.model.InstitutionalRole;
 import org.pmiops.workbench.model.Profile;
 import org.pmiops.workbench.model.VerifiedInstitutionalAffiliation;
 import org.pmiops.workbench.test.FakeClock;
 import org.pmiops.workbench.utils.mappers.CommonMappers;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
 import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
 import org.springframework.boot.test.context.TestConfiguration;
 import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Import;
+import org.springframework.context.annotation.Scope;
 import org.springframework.test.context.junit4.SpringRunner;
 
 @RunWith(SpringRunner.class)
 @DataJpaTest
 public class ProfileServiceTest {
+  private static final FakeClock CLOCK = new FakeClock(Instant.parse("2000-01-01T00:00:00.00Z"));
+
+  private static final DbInstitution BROAD_INSTITUTION =
+      new DbInstitution().setShortName("Broad").setDisplayName("The Broad Institute");
+
+  private static final VerifiedInstitutionalAffiliation BROAD_AFFILIATION =
+      new VerifiedInstitutionalAffiliation()
+          .institutionShortName("Broad")
+          .institutionDisplayName("The Broad Institute")
+          .institutionalRoleEnum(InstitutionalRole.ADMIN);
+
+  private static final Profile VALID_PROFILE =
+      new Profile()
+          .username("jdoe123")
+          .contactEmail("jdoe123@gmail.com")
+          .address(
+              new Address()
```

Comment:
It's easier to interpret and maintain if the fields look more like real data. If they're all asdf, we won't catch errors that swap them.

It's also much easier to pick up how the class is supposed to be used if there's recognizable state in it.